packages/governance/src/contractGovernance/assertions.js

```diff
@@ -41,9 +41,32 @@ const makeAssertBrandedRatio = (name, modelRatio) => {
 };
 harden(makeAssertBrandedRatio);
 
+const assertRelativeTime = value => {
+  isRemotable(value.timerBrand) || Fail`relativeTime must have a brand`;
+  typeof value.relValue === 'bigint' || Fail`must have a relValue field`;
+};
+harden(assertRelativeTime);
+
+const assertTimestamp = value => {
+  isRemotable(value.timerBrand) || Fail`timestamp must have a brand`;
+  typeof value.absValue === 'bigint' || Fail`must have an absValue field`;
+};
+harden(assertTimestamp);
+
+const makeAssertTimerService = name => {
+  return timerService => {
+    typeof timerService === 'object' ||
+      Fail`value for ${name} must be a TimerService, was ${timerService}`;
```

Comment:
is there anything more that can be checked here? if it's only asserting that the type is an object, then just just have `assertObject` instead of implying that there's any more strict validation. (I'd say to just inline the check but I see that `buildCopyParam` needs a unary function.